From fc006d4e584eed8e108db1eb9e4cc5dcd7fde7dc Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Edvinas=20Stun=C5=BE=C4=97nas?= <edvinas@8devices.com>
Date: Fri, 15 Nov 2024 23:45:54 +0000
Subject: [PATCH] clk: qcom: fix QCOM GDSC dependency for QCS404 GCC

In scenario when building kernel only for QCS40x and with single
GCC_404 clock controller enabled, GDSC regulator driver must be
enabled as well otherwise SoC peripherals won't start.

Signed-off-by: Edvinas Stunzenas <edvinas@8devices.com>
Upstream-Status: Pending

---
 drivers/clk/qcom/Kconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/clk/qcom/Kconfig b/drivers/clk/qcom/Kconfig
index a79b83758389..11247ce09682 100644
--- a/drivers/clk/qcom/Kconfig
+++ b/drivers/clk/qcom/Kconfig
@@ -402,6 +402,7 @@ config QCM_DISPCC_2290
 
 config QCS_GCC_404
 	tristate "QCS404 Global Clock Controller"
+	select QCOM_GDSC
 	help
 	  Support for the global clock controller on QCS404 devices.
 	  Say Y if you want to use multimedia devices or peripheral
